Dockerfile

```diff
@@ -24,21 +24,9 @@ RUN volta run npm ci
 COPY --link . .
 RUN volta run npm run build
 
-
-# build a file server to use for hosting the static files
-FROM rust:alpine AS build-server
-RUN apk upgrade --no-cache
-RUN apk add --no-cache musl-dev build-base
-
-ENV CARGO_HOME=/cargo
-ARG DUFS_VERSION
-RUN cargo install dufs --version $DUFS_VERSION --locked
-
-
 # host the app via the static file server
-FROM alpine:latest AS app 
-RUN apk upgrade --no-cache
+FROM sigoden/dufs
```

Comment:
Is that image an alpine image?

Reply:
That image is an [scratch](https://hub.docker.com/_/scratch) image which only contains the executable `dufs`